smartapps/samsung/sleepsense-out-of-bed.src/sleepsense-out-of-bed.groovy

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
**OVERALL COMMENT**
This functionality should be combined into the first smartapp. The listeners are already there